pallets/anchors/Cargo.toml

```
@@ -26,19 +26,23 @@ sp-runtime = { git = "https://github.com/paritytech/substrate", default-features
 sp-std = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.29" }
 
 # optional dependencies for benchmarking
+cfg-utils = { path = "../../libs/utils", default-features = false }
```

```suggestion
cfg-utils = { path = "../../libs/utils", default-features = false, optional = true }
```
We only want to compile this if we are in benchmarking